src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Set up your local build environment
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+Running the site locally makes testing and previewing large changes much easier. There are several things you need to do get this working. Fortunately, you only need to do this once.
```

Comment:
Might be overkill, but a section that covers how to open/use a terminal, and what it even is, may be helpful.